lib/chargebee/request.rb

```
@@ -1,25 +1,39 @@
+      http_url = TelemetryExecutor.build_http_url(env, sub_domain, url)
+
+      TelemetryExecutor.execute(
+        env,
+        telemetry_resource: telemetry_resource,
+        telemetry_operation: telemetry_operation,
+        method: method,
+        http_url: http_url,
+        request_headers: headers,
+      ) do |telemetry_headers|
+        merged_headers = headers.dup
+        telemetry_headers&.each { |key, value| merged_headers[key] = value }
```

RuboCop (1.88.0)

[fatal] unexpected token error
(Using Ruby 2.0 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)